src/main/resources/application.properties

```diff
@@ -6,5 +6,5 @@ spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL8Dialect
 spring.jpa.hibernate.ddl-auto=validate
 jwt.expiration=${jwt_expiration}
 jwt.secret=${jwt_secret}
-
+server.port=8081
 server.port=8081
```

Comment:
As far as I see property 'server.port' is duplicated

Reply:
done